.github/workflows/main.yml

```diff
@@ -202,6 +198,28 @@ jobs:
         with:
           name: playwright-results
           path: ./tests/sanity/playwright-report/
+      - name: Generates Allure Report
+        uses: simple-elf/allure-report-action@master
+        if: always()
```

Comment:
I think we need to enable it only for main branch

Reply:
yes, agree

Reply:
done

Reply:
Why we still use "if: always()"

Reply:
it's for generating the report, which we can also download to the local.
Deployment executing by the - name: Deploy report to Github Pages step
it used the 
if: ${{ github.ref == 'refs/heads/main' }}
param